STYLE.md

```diff
@@ -34,3 +34,10 @@ Additionally it can be installed as a commit hook with `pre-commit install`.
 - Swift code style is validated using [SwiftLint](https://github.com/realm/swiftlint)
 - The rules enforced are available in the repo's [.swiftlint.yml file](./.swiftlint.yml)
 - The linter may be run locally using `swiftlint` or auto-corrected with `swiftlint autocorrect`
+
+## Shared constructs
+
+- There's no directly supported way to universally share an enumeration across platforms. In order
+to provide some enforced consistency, we've adopted the convention of defining the enum at the
+lowest applicable layer (core/bridge) of the library, and then declaring public `extern const`
+values defined in terms of the enumeration, to be shared across bridge and platform code.
```

Comment:
@junr03 Is this what you had in mind?

Reply:
Yep this is great, could we add a link to an example?